kratos/tests/processes/test_mortar_mapper_process.cpp

```diff
@@ -29,7 +29,7 @@ namespace Kratos
     {
         typedef Node<3>                                                    NodeType;
```

Comment:
@rubenzorrilla I moved this test, to put it in a folder with the other processes test